config/locales/forms/registration_number_forms/en.yml

```diff
@@ -1,7 +1,8 @@
 en:
   registration_number_forms:
     new:
-      heading: What's the registration number of the company?
+      heading_limitedCompany: What's the registration number of the company?
+      heading_soleTrader: What's the registration number of the individual?
```

Comment:
We should never have a need for this as an individual should never go down a path that requests a registration number.

I'd rather take this out so if we have something in the sole trader journey that relies on this translation existing we know about it.

On a side note, do we have the app configured to raise an exception in dev and test if a translation is missing?

Reply:
You're completely right. I was thinking it was weird that there was a reg number page for individuals but I must have misread the wireframe/journey map. Will sort this out!

And no, we don't have that configured – I'll look into it.

Reply:
It's a little old, but I think it covers our version of rails https://robots.thoughtbot.com/foolproof-i18n-setup-in-rails

Reply:
Both these issues should now be fixed 👍